src/HRQLS/Controllers/Food/Sanitation.php

```diff
@@ -54,6 +54,10 @@ public function main(Request $req, Application $app)
                 $data['longitude'] = $toldata['coordinates']['longitude'];
```

Comment:
It's kind of outside the scope but we should be using the HerculesResponse object rather than just raw JSON.